app/views/recipes/edit.html.erb

```diff
@@ -3,6 +3,5 @@
 
   <%= render "form", recipe: @recipe %>
 
-  <%= link_to "Show this recipe", @recipe, class: "ml-2 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
-  <%= link_to "Back to recipes", recipes_path, class: "ml-2 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+  <%= link_to "Cancel", @recipe, class: "ml-2 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
```

Comment:
I think the 'Back to recipes' button in `new.html.erb` could be changed to 'Cancel' as well